src/Database/PostUpdate.php

```diff
@@ -162,8 +163,13 @@ private static function update1198()
 
 		// Set the "author-id" and "owner-id" in the item table and add a new public contact entry if needed
 		foreach ($item_arr as $item) {
-			$author_id = Contact::getIdForURL($item["author-link"]);
-			$owner_id = Contact::getIdForURL($item["owner-link"]);
+			$default = ['url' => $item['author-link'], 'name' => $item['author-name'],
```

Comment:
Local extra variables again. They make only sense when you use them more than once (IMO).

Reply:
I think it improves the readability to have a short function call and to have variables that match the meaning.